### PR TITLE
config: add suggesters for all collections

### PR DESCRIPTION
**Pull request by ammirate**
* Note: only literature and authors are tested

* INSPIR-2364

Signed-off-by: Antonio Cesarano <cesarano2607@gmail.comk>

### CODE REVIEW

File: tests/integration/test_config.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+from invenio_search import current_search_client as es
+
+
+def _get_suggester_result(response, suggester):
+    return response.json[suggester][0]['options'][0]['_source']
+
+
+def test_literature_suggesters_book_title(api_client, create_record):
+    lit = create_record("lit", data={"titles":[{"title": "Suggested title"}]})
+    es.indices.refresh('records-hep')
```
Comment (drjova) [inline tests/integration/test_config.py]: do you need to do this? 🤔 it's already on ``create_record``

File: tests/integration/test_config.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
```
Comment (drjova) [inline tests/integration/test_config.py]: I'd move this file to ``tests/integrations/config/views/test_suggesters.py`` WDYT?

File: tests/integration/test_config.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    assert result_rec_id == expected_rec_id
+
+
+def test_literature_suggesters_abstract_source(api_client, create_record):
+    lit = create_record("lit", data={"abstracts":[{"value": "Fancy abstract", "source": "WSP"}]})
+    es.indices.refresh('records-hep')
+
+    resp = api_client.get('literature/_suggest?abstract_source=ws')
+
+    result_rec_id = _get_suggester_result(resp, 'abstract_source')['control_number']
```
Comment (drjova) [inline tests/integration/test_config.py]: we should check also the other fields defined, some of the are required in the forms

File: tests/integration/test_config.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+    result = resp.json['book_title'][0]['options']
+    expected = []
+
+    assert result == expected
+
+
+@pytest.mark.xfail
+def test_author_suggesters(api_client, create_record):
+    # FIXME: author_suggest field is not populated when indexing authors
```
Comment (drjova) [inline tests/integration/test_config.py]:
```python
@pytest.mark.xfail(reason="author_suggest field is not populated when indexing authors")
```
:)

File: tests/integration/test_config.py (hunk 1 above)
Comment (ammirate) [reply tests/integration/test_config.py]: Actually nope :+1:

**Pull request merged**